static/app/components/replays/diff/replayDiffChooser.tsx

```diff
@@ -72,6 +87,10 @@ export default function ReplayDiffChooser({
   );
 }
 
+const StyledAlert = styled(Alert)`
+  margin: 0;
```

Comment:
I think it needs some top margin

Reply:
<img width="1708" alt="image" src="https://github.com/user-attachments/assets/bf9d9040-408a-432b-8dd2-0217d1f4f4a1">
does this look better to you?

Reply:
yes!